lib/internal/url.js

```diff
@@ -376,6 +377,8 @@ class URLSearchParams {
       init = toUSVString(init);
       this.#searchParams = init ? parseParams(init) : [];
     }
+
+    markAsUncloneable(this);
```

Comment:
Can you do this at the beginning of the constructor? Same below.

You are also need to do this for `URLSearchParamsIterator`.